<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageCalibWgtClass</class>
 <widget class="QWidget" name="PageCalibWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Analog inputs (ADC code)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Sup. voltage</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="cdcEngCurLbl">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_31">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Pot.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>A2 voltage</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>A1 voltage</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="adcA1VoltLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="cdcA1VoltLbl">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="adcA2VoltLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_16">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="cdcA2VoltLbl">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="adcVsupLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="cdcVsupLbl">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Full current</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="adcEngCurLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_20">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>mA</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="cdcBCurrLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="adcPotLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_23">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Temperature</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="adcTempKeyLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="QLabel" name="cdcTempKeyLbl">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="3">
          <widget class="QLabel" name="label_26">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;span style=&quot;vertical-align:super;&quot;&gt;0&lt;/span&gt; C</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>B1 voltage</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="adcB1VoltLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="cdcB1VoltLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>A current</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>B current</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="adcACurrLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="adcBCurrLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="cdcACurrLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>mA</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>mA</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>B2 voltage</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="adcB2VoltLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="cdcB2VoltLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="cdcPotLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Joy.</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="adcJoyLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="cdcJoyLbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="13" column="3">
          <widget class="QLabel" name="label_32">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item row="13" column="2">
          <widget class="QLabel" name="cdcPwmLevelLbl">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_30">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Duty</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_33">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>L</string>
           </property>
          </widget>
         </item>
         <item row="14" column="3">
          <widget class="QLabel" name="label_34">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>mΩ</string>
           </property>
          </widget>
         </item>
         <item row="15" column="3">
          <widget class="QLabel" name="label_35">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>μH</string>
           </property>
          </widget>
         </item>
         <item row="14" column="2">
          <widget class="QLabel" name="R">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="15" column="2">
          <widget class="QLabel" name="L">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>L5</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_27">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>H5</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="adcL5Lbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="adcH5Lbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="cdcL5Lbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLabel" name="cdcH5Lbl">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QLabel" name="label_18">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QLabel" name="label_28">
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
